src/app/core/services/rundown-event-observer.service.ts

```diff
@@ -43,6 +44,10 @@ export class RundownEventObserver {
     return this.eventObserver.subscribe(RundownEventType.ACTIVATED, this.createEventValidatingConsumer(onActivated, this.rundownEventParser.parseRundownActivatedEvent.bind(this.rundownEventParser)))
   }
 
+  public subscribeToRundownRehearse(onActivated: (event: RundownRehearseEvent) => void): EventSubscription {
```

Comment:
Here we subscribe to the noun so it should be `rehearsal`.

Also, it looks like you forgot to change the parameter. It should be `onRehearsal`

Reply:
Fixed